stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/content/ImageAttachmentContent.kt

```diff
@@ -86,9 +89,25 @@ public fun ImageAttachmentContent(
         if (imageCount == 1) {
             val attachment = attachments.first()
 
+            // Depending on the CDN, images might not contain their original dimensions
+            val ratio: Float? by remember(key1 = attachment.originalWidth, key2 = attachment.originalHeight) {
+                derivedStateOf {
+                    val width = attachment.originalWidth?.toFloat()
+                    val height = attachment.originalHeight?.toFloat()
+
+                    if (width != null && height != null) {
+                        width / height
+                    } else {
+                        null
+                    }
+                }
+            }
+
             ImageAttachmentContentItem(
                 attachment = attachment,
-                modifier = Modifier.weight(1f),
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .aspectRatio(ratio ?: 1f),
```

Comment:
e.g. here we can say that the ratio is "equal ratio" or something

`const val EQUAL_RATIO = 1f` for example